examples/pxScene2d/src/initGL.js

```diff
@@ -682,10 +672,16 @@ async function loadMjs(source, url, context)
             let framework = await getFile(_frameworkURL);
             vm.runInContext(framework.data, contextifiedSandbox, {filename:framework.uri});
           }
-
+          
           let file = await getFile(filename);
           let source = file.data, rpath = file.uri;
-          app = await loadMjs(source, rpath, contextifiedSandbox);
+          // define platform
+          var platformsource = "";
+          if (filename.indexOf(".mjs") != -1) {
+            let platformfile = await getFile(filename.substring(0, filename.lastIndexOf("/")+1) + "lib/lightning-spark.js");
```

Comment:
can you use _path.dirname_, _url.resolve_ please?

Reply:
addressed, please review the same